src/config/rbac/role.yaml

```diff
@@ -218,3 +218,13 @@ rules:
   - patch
   - update
   - watch
+- apiGroups:
+  - apps
+  - ""
+  resources:
+  - deployments
+  - services
+  - nodes
```

Comment:
Could we donot combine the different api groups?

The benefits are:
1. The style of this file is unified.
2. #103 will have a much easier conflict to resolve.